hail/python/hail/__init__.py

```diff
@@ -11,6 +11,18 @@
 del pkg_resources
 del sys
 
+try:
+    is_notebook = get_ipython().__class__.__name__ == 'ZMQInteractiveShell'
```

Comment:
One query, one comment:

Query: how quick is `time python3 -c 'import hail as hl'` on this branch and on `main`?

Comment: we should re-use this variable instead of calling the same code in hailtop.utils.rich_progress_bar

Reply:
in `zsh` on my machine with `n = 5`, the difference between the execution time on this branch and `main` was `-0.14s`; no reason it should be faster than `main`, so i assume that's just due to natural variation

in a jupyter notebook with the same parameters the difference was `0.12s`

good catch, didn't realize we already checked whether we were in a notebook somewhere; updated! i'm not super clear on how `hailtop.utils.rich_progress_bar.is_notebook` is currently used, though. what would be a good example to try out both locally and in a notebook to make sure that it still works as expected with the change?